app/models/item.rb

```diff
@@ -0,0 +1,3 @@
+class Item < ApplicationRecord
+  validates :name, presence: true
```

Comment:
Maybe good idea to have Uniqe item? Not sure how item will work and what purpose of it